docs/languages/en/ref/overview.rst

```diff
@@ -59,4 +59,4 @@ available to address it.
 .. _`HTML5 form rendering`: http://www.w3.org/TR/html5/forms.html#forms
 .. _`Zend Technologies`: http://www.zend.com
 .. _`mailing lists`: http://framework.zend.com/archives
-.. _`IRC channels`: http://www.zftalk.com
```

Comment:
The IRC channel is quite active, but clearly the domain is being used for something else. We should change this to `irc://irc.freenode.net/zftalk`